.gitignore

```diff
@@ -92,4 +92,5 @@ iOSInjectionProject/
 
 # System files
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+Healthy/Resources/Generated
```

Comment:
```suggestion

# Ignore generated code
*/*/Generated
```

Not sure if this will work but it's better than using specific destination.

Reply:
Also, we need to consider regeneration of the files in the workflow, it fails 👀

Reply:
This might be the most important change in the PR... LMK if you need any assistance.

Reply:
I'll try that wildcard pattern. 

Let the workflow issue to me, I'll handle that.